docs/content/TextInput.mdx

```diff
@@ -91,43 +178,70 @@ TextInput is a form component to add default styling to the native text input.
     name="block"
     type="boolean"
     defaultValue="false"
-    description={
-      <>
-        Creates a full width input element
-      </>
-    }
+    description={<>Creates a full width input element</>}
   />
   <PropsTableRow
     name="contrast"
     type="boolean"
     defaultValue="false"
     description="Changes background color to a higher contrast color"
   />
-  <PropsTableRow name='size' type="'small' | 'medium' | 'large'"     description="Creates a smaller or larger input than the default." />
-
-<PropsTableRow
-  name="leadingVisual"
-  type={<>string | React.ComponentType</>}
-  description="Visual positioned on the left edge inside the input"
-/>
-<PropsTableRow
-  name="trailingVisual"
-  type={<>string | React.ComponentType</>}
-  description="Visual positioned on the right edge inside the input"
-/>
-<PropsTableRow
-  name="validationStatus"
-  type="'error' | 'success' | 'warning'"
-  description="Style the input to match the status"
-/>
-
-<PropsTableRow
-  name="variant"
-  type="'small' | 'medium' | 'large'"
-  description="(Use size) Creates a smaller or larger input than the default."
-  deprecated
-/>
-
+  <PropsTableRow
+    name="size"
+    type="'small' | 'medium' | 'large'"
+    description="Creates a smaller or larger input than the default."
+  />
+  <PropsTableRow name="isLoading" type="boolean" description="Whether the to show a loading indicator in the input" />
+  <PropsTableRow
+    name="loaderPosition"
+    type={"'auto' | 'leading' | 'trailing'"}
+    description={() => (
+      <>
+        <div>Which position to render the loading indicator</div>
+        <ul>
+          <li>
+            'auto' (default): at the end of the input, unless a `leadingVisual` is passed. Then, it will render at the
+            beginning
+          </li>
+          <li>'leading': at the beginning of the input</li>
+          <li>'trailing': at the end of the input</li>
+        </ul>
+      </>
+    )}
```

Comment:
The description doesn't show up in the deployed docs? 🤔

Reply:
Fixed